### PR TITLE
ci(renovate): replace dependabot with renovate

### PR DESCRIPTION
**Pull request by michalvankodev**
Replace dependabot with renovate. 

- mainly for the automerge feature.

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["config:base"],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch"],
+      "matchCurrentVersion": "!/^0/",
+      "automerge": true
+    }
+  ]
```
Comment (poikaa) [inline .github/renovate.json:9]:
```suggestion
  ],
  "ignoreTests": true
```

> ### Absence of tests
> By default, Renovate will not automerge until it sees passing status checks / check runs for the branch. If you have no tests but still want Renovate to automerge, you need to add "ignoreTests": true to your configuration. However, we strongly recommend you have tests in any project where you are regularly updating dependencies.

https://docs.renovatebot.com/key-concepts/automerge/

File: package-lock.json [hunk 2]
```diff
@@ -8199,6 +8199,11 @@
         "safer-buffer": "^2.0.2",
         "tweetnacl": "~0.14.0"
       },
+      "bin": {
+        "sshpk-conv": "bin/sshpk-conv",
+        "sshpk-sign": "bin/sshpk-sign",
+        "sshpk-verify": "bin/sshpk-verify"
+      },
```
Comment (poikaa) [inline package-lock.json:8206]: Intended? 👀

**Review by poikaa — COMMENTED** [review]
> ### Codeowners
> Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.

https://docs.renovatebot.com/key-concepts/automerge/

I solved this by excluding `package.json` and `package-lock.json` in `CODEOWNERS` 👉 &nbsp; [aws-ssm-cli/CODEOWNERS](https://github.com/sudolabs-io/aws-ssm-cli/blob/main/.github/CODEOWNERS)

File: package-lock.json (hunk 2 above)
Comment (michalvankodev) [reply package-lock.json:8206]: Have I been pawned?

I just installed dependencies on my machine with `npm i`

Comment (poikaa) [reply package-lock.json:8206]: Interesting 🤔 . `sshpk` version `1.16.1` was published 3 years ago. NPM doesn't allow you to publish the same version twice, so I wouldn't consider this change as a threat.

**Review by poikaa — APPROVED** [review]

**Pull request merged**